server/lib/search.js

```diff
@@ -143,6 +152,12 @@ export const searchCollectivesInDB = async (
     dynamicConditions += `AND "countryISO" IN (:countryCodes) `;
   }
 
+  if (tags?.length) {
+    tags.forEach(tag => {
+      dynamicConditions += `AND '${tag}' = ANY("tags") `;
```

Comment:
:no_entry_sign: Never directly replace, that's the door for SQL injections. If you query this with something like:

```gql
search(tag: "xxx' = 'yyy' OR collective.data."anyPrivateField" LIKE '%bruteforce%' AND 'xxx") {
  ...
}
```

You'll have a nice surprise. 

Also, the Postgres containment operator seems like a better fit here:

```sql
# Will returns all collectives whose tags contains the array searched_tags
WHERE collective.tags @> :searched_tags
```